megamek/src/megamek/client/ui/swing/lobby/ChatLounge.java

```diff
@@ -181,84 +163,91 @@ public class ChatLounge extends AbstractPhaseDisplay implements
+    private final JPanel panMapButtons = new JPanel();
+    private final JLabel lblBoardsAvailable = new JLabel();
     private JList<String> lisBoardsAvailable;
-    private JButton butSpaceSize = new JButton(Messages.getString("ChatLounge.MapSize"));
+    private final JButton butSpaceSize = new JButton(Messages.getString("ChatLounge.MapSize"));
     boolean resetAvailBoardSelection = false;
     boolean resetSelectedBoards = true;
     private ClientDialog boardPreviewW;
-    private Game boardPreviewGame = new Game();
+    private final Game boardPreviewGame = new Game();
     private transient BoardView previewBV;
     Dimension currentMapButtonSize = new Dimension(0, 0);
     private final JCheckBox showPlayerDeployment = new JCheckBox(Messages.getString("ChatLounge.showPlayerDeployment"));
 
-    private ArrayList<String> invalidBoards = new ArrayList<>();
-    private ArrayList<String> serverBoards = new ArrayList<>();
+    private final ArrayList<String> invalidBoards = new ArrayList<>();
+    private final ArrayList<String> serverBoards = new ArrayList<>();
 
     private JSplitPane splGroundMap;
-    private JLabel lblSearch = new JLabel(Messages.getString("ChatLounge.labSearch"));
-    private JTextField fldSearch = new JTextField(10);
-    private JButton butCancelSearch = new JButton(Messages.getString("ChatLounge.butCancelSearch"));
+    private final JLabel lblSearch = new JLabel(Messages.getString("ChatLounge.labSearch"));
+    private final JTextField fldSearch = new JTextField(10);
+    private final JButton butCancelSearch = new JButton(Messages.getString("ChatLounge.butCancelSearch"));
 
     private transient MekTableSorter activeSorter;
     private transient ArrayList<MekTableSorter> unitSorters = new ArrayList<>();
     private transient ArrayList<MekTableSorter> bvSorters = new ArrayList<>();
 
-    private JButton butAddY = new JButton(Messages.getString("ChatLounge.butAdd"));
-    private JButton butAddX = new JButton(Messages.getString("ChatLounge.butAdd"));
-    private JButton butSaveMapSetup = new JButton(Messages.getString("ChatLounge.map.saveMapSetup") + " *");
-    private JButton butLoadMapSetup = new JButton(Messages.getString("ChatLounge.map.loadMapSetup"));
+    private final JButton butAddY = new JButton(Messages.getString("ChatLounge.butAdd"));
+    private final JButton butAddX = new JButton(Messages.getString("ChatLounge.butAdd"));
+    private final JButton butSaveMapSetup = new JButton(Messages.getString("ChatLounge.map.saveMapSetup") + " *");
+    private final JButton butLoadMapSetup = new JButton(Messages.getString("ChatLounge.map.loadMapSetup"));
 
     /* Team Overview Panel */
     private TeamOverviewPanel panTeamOverview;
     JButton butDetach = new JButton(Messages.getString("ChatLounge.butDetach"));
-    private JSplitPane splitPaneMain;
+    private final JSplitPane splitPaneMain;
```

Comment:
All those properties could be made final, so I made them final, and as a bonus sonarqube and intelliJ stopped yelling at me that those were not final.